docs/docsite/rst/become.rst

```diff
@@ -202,8 +205,63 @@ or '/bin/chmod' as the allowed commands this will fail with ansible as those
 paths won't match with the temporary file that ansible creates to run the
 module.
 
+.. _become-network:
+
+Become and Networks
+===================
+
+Ansible 2.5 added support for ``become`` to be used to enter `enable` mode (Privileged EXEC mode) on network devices that support it. This replaces the previous ``authorize`` and ``auth_pass`` options in ``provider``.
+
+This functionality requires the host connection type to be using ``connection: network_cli``, in Ansible 2.5 this is limited to ``eos`` and ``ios``.
+
+This allows privileges to be raised for the specific tasks that need them. Adding ``become: true`` and ``become_method: enable`` informs Ansible to go into privilege mode before executing the task.
+
+If a task fails with the following then it's an indicator that `enable` mode is required:
+
+.. code-block:: console
+
+   Invalid input (privileged mode required)
+
+Which can be enabled as shown:
```

Comment:
might be worthwhile to note the `beocme` and `become_method` can specific at the play level as opposed to the task level so that it can be specified once and not have to be specified for every task

Reply:
Good point, updated